lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -1507,7 +1507,7 @@ bool hasLoopMetadata(const Module *M) {
       const Instruction *Term = BB.getTerminator();
       if (!Term)
         continue;
-      if (const MDNode *MD = Term->getMetadata("llvm.loop"))
+      if (Term->getMetadata("llvm.loop"))
```

Comment:
Maybe to rewrite this as:
`
const Instruction *Term = BB.getTerminator();
if (Term && Term->getMetadata("llvm.loop"))
  return true;
`